src/sockperf.cpp

```diff
@@ -1718,7 +1718,16 @@ static int resolve_sockaddr(const char *host, const char *port, int sock_type,
     int res = getaddrinfo(host, port, &hints, &result);
     if (res == 0) {
         get_socket_address(result, addr, addr_len);
+    } else if (res == EAI_NONAME && host != NULL) { // Unix Domain Socket name
```

Comment:
Added "unix:" prefix as requested 
attaching screenshots for both cases if  --client_addr is provided or not 

![unix prefix with specific binding path](https://user-images.githubusercontent.com/44922030/167402290-4f90c6d1-33cc-477e-b9bd-bfa582bacdcf.JPG)
![unix prefix](https://user-images.githubusercontent.com/44922030/167402295-dfda5b81-3866-4ed5-b5e2-ebee377e00a8.JPG)

Reply:
wait, I understand the request to have "unix:" be in the user interface for the --addr <string>
but remove it from the UDS sysfs file path

Reply:
@rosenbaumalex, yes I mean it.

I just realized that adding "unix:" will make feedfile parser more complex because `:` is a field separator. So I suggest to switch to another idea and force user to specify a full path to unix socket (starting with /).

@EldarShalev, please check that UNIX socket names is supported in feedfile parser (function `set_sockets_from_feedfile`).

Reply:
do you consider any special symbol at the start excluding `.:[0-9][a-f][A-F]` to simplify address ip4/ip6/unix recognition?

Reply:
I thought about this regex for unix domain socket feedfile 
```
#define UNIX_DOMAIN_SOCKET_FORMAT_REG_EXP                                                          \
        "^[UuTt]:[Uu][Nn][Ii][Xx]\\/([0-9a-zA-z\\/]+)[\r\n]"

```

Reply:
I suggest `"^[UuTt]:[Uu][Nn][Ii][Xx]/(.+)[\\r\\n]"` because any symbol may be a part of socket name. Escaping is needed for `\r` and `\n` and not needed for `/`.

Reply:
I meant something as `~unix_socket_name` where `~` is the symbol that can not be a part of ipv4/ipv6 address as an alternative to `unix`

Reply:
`/` at the start of name is enough to distinguish between file name and ip address or host name

Reply:
I am fine with any symbol (it might be `/`) that allows to simplify ipv4/ipv6/unix recognition.